types/entities.ts

```diff
@@ -38,6 +38,33 @@ export interface User {
   bioHtml: string | null;
 }
 
+export interface SeriesWorkSummary
+  extends Omit<
+    WorkSummary,
+    "category" | "publishedAt" | "rating" | "tags" | "stats" | "locked"
+  > {
+  tags: Omit<WorkSummary["tags"], "warnings">;
+  stats: Omit<WorkSummary["stats"], "comments">;
+}
+
+export interface Series {
+  id: string;
+  name: string;
+  begunAt: string;
+  updatedAt: string;
+  creators: WorkSummary["authors"];
```

Comment:
What about `authors` for consistency, instead of `creators`? If they're different concepts, let me know!

Reply:
I can rename it! I originally named it `authors`, then changed it to `creators` after reading over #36 again and looking at the series index page.

Reply:
Oh, I see! Yeah, I think authors works best.

We had also started doing some work on the `Series` type here, but lost steam: https://github.com/Fandom-Coders/AO3.js/pull/50 

I'm ok with your types as they are (with the corrections), but I thought I'd show you in case there's things you want to fold in.

Reply:
Sounds good! Thanks for letting me know!